assembly/std/context.ts

```diff
@@ -83,7 +79,7 @@ export function caller(): Address {
  *
  * Callee is the current smart-contract address.
  *
- * @return {Address} returns the stack element at the top of the address stack
+ * @returns the stack element at the top of the address stack
```

Comment:
same. the callee concept is maybe unclear. You can precise it by adding a synonym such as current smart contract address ?